packer/windows.pkr.hcl

```diff
@@ -76,6 +76,7 @@ source "googlecompute" "windows" {
   winrm_insecure          = true
   winrm_use_ssl           = true
   winrm_timeout           = "10m"
+  state_timeout           = "10m"
```

Comment:
Hm. Do you suspect this will actually help? 

Seems to be consistently taking ~3min in the ~5 cases runs I looked at.

I'd normally prefer to do something like this in a PR with a different title than "ci: Deprecate images on failure" :)